dev_guide/deployments.adoc

```diff
@@ -138,16 +170,28 @@ $ osc describe dc <deployment_config>
 ----
 
 == Canceling a Deployment
+You might want to cancel a deployment if:
+
+* A deployment was initiated incorrectly, and you want to cancel it before it can do further damage.
+* There is a security fix you need to apply, and you do not want to wait for an ongoing deployment to complete.
+* A deployment or hook implementation is stuck after running for too long, and you cannot initiate a deployment when there is a deployment already running.
+
+[NOTE]
+====
+By default, the deployment process is restricted to 6 hours. If a deployment is
+still in progress after 6 hours, then the containers for the deployment pod and
+any deployment hook pods are all deleted, and these pods are placed in the
+Failed state.
+====
 
-To cancel a running deployment:
+Cancel a running or stuck deployment with the following command:
 
 ----
-$ oc deploy <deployment_config> --cancel
+$ oc deploy <config_name> --cancel
 ----
 
-WARNING: The cancelation is a best-effort operation, and may take some time to
-complete. It's possible the deployment will partially or totally complete
-before the cancelation is effective.
+After running this command, the deployment eventually shuts down and transitions
+to a Failed status.
```

Comment:
I don't think I like the "Failed status" part since deployment "states" or "statuses" aren't described anywhere, making this seem a bit out of context. Do you think the detail provided in my warning is irrelevant?